### ISSUE TITLE
Attach file object on error object

### ISSUE THREAD

**Issue by heinzmuller** [shannonmoeller/gulp-hb#23]
I don't know if this is the correct way of doing this, but I've done some testing locally and this seems to work totally fine.
```
catch (err) {
	// Attach file to error object
	err.file = file;
	cb(new gutil.PluginError('gulp-hb', err, {
		stack: err.stack,
		showStack: true
	}));
}
```
This allows you to grab the failed file when handling errors
```
...
.pipe(plumber({errorHandler: notify.onError("<%= error.file.relative %>: <%= error.message %>")}))
.pipe(hb())
...
[14:22:57] gulp-notify: [Error running Gulp] search.hbs: Parse error on line 5:
```

**Comment by shannonmoeller** [comment]
I have a v3 release candidate ready that should address this issue. Are you available to kick the tires?

    npm i --save "shannonmoeller/gulp-hb#v3"

**Comment by shannonmoeller** [comment]
Released as v3.0.0.

**Comment by heinzmuller** [comment]
Sorry for the late reply, thanks for the addition!

I'll give 3.0 a go :)
